rules/xcodeproj.bzl

```diff
@@ -116,6 +116,7 @@ def _xcodeproj_aspect_impl(target, ctx):
             if test_host_target:
                 test_host_appname = test_host_target[_TargetInfo].direct_targets[0].name
 
+        framework_includes = depset([], transitive = _get_attr_values_for_name(deps, _SrcsInfo, "framework_includes"))
```

Comment:
just curious if moving here makes any diff. or it's just to make code clenaer

Reply:
That did not make a difference. I think it does make code slightly cleaner.